tests/Microsoft.Graph.Test/Models/Extensions/ContentConverterExtensionsTests.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Microsoft.Graph.Extensions;
+using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.Graph.DotnetCore.Core.Test.Extensions
+{
+    [TestClass]
```

Comment:
I thought we had started migrating all the tests to XUnit?

Reply:
The project Microsoft.Graph.Test no longer exists, the test are now under  Microsoft.Graph.DotnetCore.Test > Models > Extensions

Reply:
You'll need to merge dev into your branch to capture the fact that we no longer use mstest.

Reply:
@titusgicheru Since you've moved this MSTest test to ` Microsoft.Graph.DotnetCore.Test > Models > Extensions` as an XUnit test, you should subsequently remove it from this PR.